tests/schema/tests.py

```diff
@@ -4861,6 +4861,24 @@ def test_add_db_comment_charfield(self):
             comment,
         )
 
+    @skipUnlessDBFeature("supports_comments")
```

Comment:
Some databases don't support stored generated columns
```suggestion
    @skipUnlessDBFeature("supports_comments", "supports_stored_generated_columns")
```

Reply:
Thank you! I have made the corrections